drivers/st25r3911b/Kconfig

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig ST25R3911B_LIB
+	bool "NFC ST25R3911B library"
+	help
+	  Enable the NFC ST25R3911B library
+
+if ST25R3911B_LIB
+
+config ST25R3911B_SPI_FREQ
+	int "ST25R3911B SPI frequency"
+	default 4000000
+	help
+	  ST25R3911B SPI frequency
+
+config ST25R3911B_SPI_PORT
+	int "ST25R3911 SPI port"
+	default 0
+	help
+	  Configure a SPI port for ST25R3911B
+
+config ST25R3911B_IRQ_PIN
+	int "ST25R3911B IRQ pin"
+	default 3
+	help
+	  Configure an IRQ pin for ST25R3911B
```

Comment:
And this must be a pin from GPIO port 0. This limitation should be documented.
Ideally, this configuration (all 3 entries) should be moved to DeviceTree (then the user would be able to use a pin from port 1 as well).

Reply:
I added also the port configuration. In future this can be added to DeviceTree, PR with changes is in progress.